packages/datagateway-dataview/src/index.tsx

```diff
@@ -171,62 +171,36 @@ if (
   render();
   log.setDefaultLevel(log.levels.DEBUG);
 
-  if (process.env.NODE_ENV === `development`) {
-    settings.then((settingsResult) => {
-      if (settingsResult) {
-        const splitUrl = settingsResult.downloadApiUrl.split('/');
-        const icatUrl = `${splitUrl
-          .slice(0, splitUrl.length - 1)
-          .join('/')}/icat`;
-        axios
-          .post(
-            `${icatUrl}/session`,
-            `json=${JSON.stringify({
-              plugin: 'simple',
-              credentials: [{ username: 'root' }, { password: 'pw' }],
-            })}`,
-            {
-              headers: {
-                'Content-Type': 'application/x-www-form-urlencoded',
-              },
-            }
-          )
-          .then((response) => {
-            axios
-              .get(`${settingsResult['apiUrl']}/sessions`, {
-                headers: {
-                  Authorization: `Bearer ${response.data.sessionId}`,
-                },
-              })
-              .then(() => {
-                const jwtHeader = { alg: 'HS256', typ: 'JWT' };
-                const payload = {
-                  sessionId: response.data.sessionId,
-                  username: 'Thomas409',
-                };
-                const jwt = jsrsasign.KJUR.jws.JWS.sign(
-                  'HS256',
-                  jwtHeader,
-                  payload,
-                  'shh'
-                );
-
-                window.localStorage.setItem(MicroFrontendToken, jwt);
-              })
-              .catch((error) => {
-                log.error(
-                  `datagateway-api cannot verify ICAT session id: ${error.message}.
-                     This is likely caused if datagateway-api is pointing to a
-                     different ICAT than the one used by the IDS/TopCAT`
-                );
-              });
-          })
-          .catch((error) =>
-            log.error(`Can't log in to ICAT: ${error.message}`)
+  settings.then((settingsResult) => {
+    if (settingsResult) {
+      const splitUrl = settingsResult.apiUrl.split('/');
+      const dataGatewayUrl = `${splitUrl
+        .slice(0, splitUrl.length - 1)
+        .join('/')}/datagateway-api`;
```

Comment:
Can't we just use `apiUrl` from settings? We don't need to split and then add on `datagateway-api` again - we only had to do that previously because often TopCAT and ICAT were on the same server in dev, so you could derive the ICAT url from the TopCAT url (`downloadApiUrl`)

Reply:
Yes, this is much better thanks. Changes made.